### ISSUE TITLE
Update memilio-simulation setup.py and documentation

### ISSUE THREAD

**Issue by AgathaSchmidt** [DLR-SC/memilio#295]
When installing the memilio python bindings there are some packages like pandas missing. You need to run the setup.py of memilio-epidata first. If the intention is that you can set up these two independently they should each have complete setup.py 's .

**Comment by dabele** [comment]
I'm surprised the memilio-simulation package requires pandas

**Comment by AgathaSchmidt** [comment]
I think I mixed something up in the issue description. 
The goal was to run the **pycode/examples/simulation/secir_simple** file. 
It was unclear which setup.py has to be executed. Intuitively the memilio-simulation setup.py comes into mind. But without running the memilio-epidata setup.py not all packages needed are installed. 
Maybe it could be sufficent to add a ReadMe to the pycode/examples folder.

**Comment by dabele** [comment]
I don't think we should install pandas with the python bindings just so a user can run the examples, even the examples that specifically demonstrate the use of the python bindings. Note that the examples are not installed with the package. I agree there could be a hint somwhere. Ideally the examples are mentioned in the README of pycode or pycode/memilio-simulation. Adding a README to pycode/examples may be enough.
